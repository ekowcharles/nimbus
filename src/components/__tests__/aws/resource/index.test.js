import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Resource from '../../../aws/resource/index'

describe('Resource', () => {
  it('renders resources', () => {
    const wrapper = shallow(
      <Resource
        hideName={false}
        hideStatus={false}
        image="db.png"
        name="...000001"
        state="available"
      />
    )

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})