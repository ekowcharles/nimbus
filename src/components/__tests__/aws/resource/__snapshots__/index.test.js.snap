// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resource renders resources 1`] = `
<div
  className="res-container clickable"
  onClick={[Function]}
>
  <ResourceStatus
    hide={false}
    state="available"
  />
  <img
    alt="resource"
    className="res-container-icon"
    src="db.png"
  />
  <span>
    ...000001
  </span>
</div>
`;
